Readme.md

```diff
@@ -1,59 +1,60 @@
-![](https://img.shields.io/github/contributors-anon/byte-physics/igor-unit-testing-framework?style=plastic)
-![](https://img.shields.io/github/license/byte-physics/igor-unit-testing-framework?style=plastic)
+[![Contributors](https://img.shields.io/github/contributors-anon/byte-physics/igor-unit-testing-framework?style=plastic)](https://github.com/byte-physics/igor-unit-testing-framework/graphs/contributors)
+[![GitHub release (latest by date)](https://img.shields.io/github/v/release/byte-physics/igor-unit-testing-framework?style=plastic)](https://github.com/byte-physics/igor-unit-testing-framework/releases)
+[![License](https://img.shields.io/github/license/byte-physics/igor-unit-testing-framework?style=plastic)](https://github.com/byte-physics/igor-unit-testing-framework/blob/main/License.txt)
 
-This project aims at providing a complete set of tools for programmers for writing and maintaining unit tests.
+The Igor Pro Universal Testing Framework (IUTF) is a versatile tool to write and run tests for [Igor Pro](https://www.wavemetrics.com/products/igorpro) code.
+The results of the tests is outputted in standardized formats that can be parsed by common
+[CI](https://en.wikipedia.org/wiki/Continuous_integration) frameworks. Test runs can also be fully automated as part of a CI/CD workflow.
+For simpler usage scenarios a summary of the test run is shown in the history after it finished.
 
```

Comment:
Can we split the following part in a new section? The current one seems a bit long. The new section could be something like "What is IUTF?" or "What can IUTF be used for?"